notebooks/official/migration/UJ1 Vertex SDK AutoML Image Classification.ipynb

```diff
@@ -0,0 +1,1432 @@
+{
```

Comment:
<p>I'm not sure if DAG would be the right term, since the variable just represents the training job and not a pipeline.</p><p> </p><p>In any case, the term DAG may be ambiguous or new to users, even if it was accurate.</p><p> </p><p>Recommend just using the term "job" in text and code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
I will have to re-review the GAPIC versions. If I recall, the AutoML training creates a "AutoML-specific" pipeline. For custom, you could either do a job or use the same pipeline.

Anyways, if its called a pipeline, users expect the output is conceptually a graph (regardless of actual brain-dead implementation :). 

After my review, I will make revisit text/code.

Reply:
Haha okay, the point of the MBSDK is to abstract that strange concept of a TrainingPipeline away from the user. That way they don't even think about a pipeline. Of course, if they looked in the console, it would show a pipeline, but there it is.

I would just leave with a comment that not every user will know what a DAG is and why it's relevant in that cell.

Reply:
I can define DAG on first use.

Reply:
updated